Apromore-Core-Components/Apromore-BPMN-Editor/Apromore-BPMN-Modeler-JS/README.md

```diff
@@ -0,0 +1,38 @@
+# QBPSimulator
+Integration of the QBP process simulator into Apromore
```

Comment:
Can we update the top-level introduction of this module to provide more clarity? My understanding is: "This module is to build a distribution of bpmn.io editor including custom extensions for Apromore. These extensions could be developed in-house like the Simulator or from bpmn.io/third-party sources like the Color Picker.".